spec/requests/api/v1/provider_spec.rb

```diff
@@ -80,7 +80,7 @@ def get_provider_codes_from_body(body)
         build(:ucas_preferences,
               type_of_gt12: :coming_or_not,
               send_application_alerts: :none,
-              application_alert_email: 'application_alert_recipient@acmeuniversity.education.uk')
+              application_alert_email: nil)
```

Comment:
Why remove this from the test? Would it make sense to test this instead of the `nil` case?

Reply:
In this test two providers are parsed into JSON. Of these two, one has an application alert recipient and one doesn't. So it now tests both cases in one test.